assets/css/_side.scss

```diff
@@ -0,0 +1,49 @@
+@import '_variables';
```

Comment:
no need for multiple import of variable and mixin , just import in style.scss one time